packages/devextreme/js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -199,12 +199,13 @@ export class ResizingController extends modules.ViewController {
     const totalItemsCount = Math.max(0, this._dataController.totalItemsCount());
     const widgetAriaLabel = this._getWidgetAriaLabel();
     const widgetStatusText = messageLocalization
-      // @ts-expect-error Badly typed format method
+    // @ts-expect-error Badly typed format method
       .format(widgetAriaLabel, totalItemsCount, columnCount);
     // @ts-expect-error Badly typed dxElementWrapper
     const $ariaLabelElement = this.component.$element().children(`.${GRIDBASE_CONTAINER_CLASS}`);
-
-    this.component.setAria('label', widgetStatusText, $ariaLabelElement);
+    // @ts-expect-error Treelist Variable
+    const expandableWidgetAriaLabel = messageLocalization.format(this._expandableWidgetAriaId);
+    this.component.setAria('label', `${widgetStatusText}. ${expandableWidgetAriaLabel}`, $ariaLabelElement);
```

Comment:
There are different ways to build the description. To avoid changing in the existing test you could use the following:
```
const labelParts = [widgetStatusText];
if (expandableWidgetAriaLabel) {
    labelParts.push(expandableWidgetAriaLabel);
}
this.component.setAria('label', labelParts.join['. ']);
```

Reply:
Or
const labelParts = [
    // @ts-expect-error Badly typed format method
    messageLocalization .format(widgetAriaLabel, totalItemsCount, columnCount),
    messageLocalization.format(this._expandableWidgetAriaId),
];
const label =  labelParts.filter((x) => x ).join['. '];
this.component.setAria('label', label, $ariaLabelElement);